pkg/render/node_test.go

```diff
@@ -45,6 +45,8 @@ var _ = Describe("Node rendering tests", func() {
 	one := intstr.FromInt(1)
 	defaultNumExpectedResources := 6
 
+	k8s := render.K8sServiceEndpoint{}
```

Comment:
just a nit, but can we call this something other than "k8s" - it's not a particularly descriptive name, and lots of stuff here is relevant to Kubernetes!

Other options that come to mind:

- kubernetesServiceEndpoint
- k8sAPIInfo
- apiServerLocation
- ? ? ?

Reply:
sure k8sServiceEp ?